api-docs/rst/dev-guide/glossary.rst

```diff
@@ -36,14 +36,14 @@ Glossary 
     is mapped directly to an individual computer or a virtual machine.
 
 **Convergence**
-    The act of Auto Scale adding or removing enough servers to satisfy
-    the needed capacity.
+    The act of Auto Scale adding or removing enough units to
+    satisfy the needed capacity.
```

Comment:
The act of Auto Scale adding or removing scaling units to meet the capacity requirements specified in the Autoscale configuration. 

So capacity is defined by configurations and policies?    

could be:
specified in the ... 
configurations
policy
policies
configuration and policies?  

I think it might be policies ....